tmol/score/omega/potentials/omega.hh

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <Eigen/Core>
+#include <cmath>
+
+#include <tmol/score/common/cubic_hermite_polynomial.hh>
+#include <tmol/score/common/tuple.hh>
+
+namespace tmol {
+namespace score {
+namespace omega {
+namespace potentials {
+
+#define def auto EIGEN_DEVICE_FUNC EIGEN_STRONG_INLINE
+
+using namespace tmol::score::common;
```

Comment:
I would like to avoid "using namespace" statements in header files

Reply:
Huh, I think I coped it from another header file that I was using as an example. I've moved it to compiled.ops.cpp where it is needed.